microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-xml-maven.adoc

```diff
@@ -0,0 +1,73 @@
+= Microsoft .Net Connector 3.1 XML and Maven Support - Mule 4
```

Comment:
```suggestion
= Microsoft .NET Connector 3.1 XML and Maven Support - Mule 4
```
Also, note "XML and Maven Support" rather than early ref to "XML Maven".